cpp/oneapi/dal/algo/svm/backend/cpu/train_kernel.cpp

```diff
@@ -53,12 +60,9 @@ static train_result call_daal_kernel(const context_cpu& ctx,
     const auto daal_labels  = interop::convert_to_daal_homogen_table(arr_label, row_count, 1);
     const auto daal_weights = interop::convert_to_daal_homogen_table(arr_weights, row_count, 1);
 
-    // TODO: move as parameter onedal SVM
-    auto kernel =
-        daal_kernel_function::KernelIfacePtr(new daal_kernel_function::linear::Batch<Float>());
-
+    const auto daal_kernel = desc.get_kernel_impl()->get_impl()->get_interop_kernel();
```

Comment:
Probably, its better to return from impl not a interop kernel, but `daal_svm::Parameter`. I think descriptor have all information to create it.

Reply:
You mean
```c++
auto daal_parameter = desc.get_impl()->get_interop();
```
It will not be strange in comparison with other algorithms?

Reply:
I think we can discuss this question later - it's ok for now